src/elements/common/base.scss

```diff
@@ -12,6 +12,20 @@
     line-height: 20px;
     letter-spacing: .3px;
 
+    // TODO: Remove when border-color in src/styles/_inputs.scss is accessible
```

Comment:
When will this be? We can't we update it there now?

Reply:
tbh I'm not sure if we'll ever update that stylesheet before we migrate everything to ui-toolkit.

The reason we're not modifying it now is because it'll be a global change and design recommended to only update the ones in question. the other inputs would be fixed with the migration to ui-toolkit. I think this is because updating the border now won't align with the current text color and placeholder color for those inputs. i.e. the new border color will look weird with the old text color so we'd need to update all of them together with all the different interaction states

Reply:
That makes sense, thanks for explaining!